onnxruntime/python/tools/transformers/fusion_attention_clip.py

```diff
@@ -320,7 +320,7 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             output=attention_last_node.output[0],
             add_qk_str=add_qk,
             scale=None,
-            causal=(add_mask is not None),
+            causal=False,
```

Comment:
Need to set based on the causal mask matching. 

For example, lines 291 to 304 match causal mask. If it is matched, this shall be set as True.